script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_4.txt

```diff
@@ -117,5 +117,5 @@ This is bad. This can't even be compared to what happened at the hotel. If he ap
 "Arcueid...!"
 "I know. We need to hurry, Shiki."
 <Page35>
-I start running toward the station.
+I start running towards the station.
```

Comment:
I think I like toward more... but I don't really know...

Reply:
they seem to be equivalent but toward is American, so I guess we can standardize on that